_docs/sdk/android.md

```diff
@@ -29,70 +41,177 @@ allprojects {
 }
 ```
 
-Also add this line to your module gradle: `implementation('com.smartlook.recording:app:0.9.0.2.5.3-beta')`
-
 ## API Reference
 
 Applications can interact with the SDK using public SDK methods.
 
 ### Initialize Smartlook
 
-You must supply your **Smartlook SDK key** which can be acquired in the Smartlook Dashboard.
+You need to provide your **Smartlook SDK Key** which can be acquired in the [Smartlook Dashboard](https://www.smartlook.com/app/dashboard/settings/projects){:target="_blank"}.
+{: .alert .alert-warning }
+
+For standard initialization use: 
+
+```Java
+Smartlook.init(String YOUR_API_KEY)
+```
+
+This method initializes the SDK **and starts the recording.** Best place to call this method is Application class or any entry point to your app.
 
-Basic init method is `Smartlook.init(YOUR_API_KEY)` This method initializes the SDK **and starts the recording.** Best place to call this method is Application class or any entry point to your app.
+#### Initalize irregular
```

Comment:
This is a weird title, let's think of something else.

Reply:
Created a trello card and we gonna discus this topic on ` s y n c ` call with iOS team.